app/models/concerns/requests/voyager.rb

```diff
@@ -63,7 +63,7 @@ def param_mapping(bib, user, item)
     end
 
     def request_payload(item, parameter_name: "recall-parameters", expiration_period: 60)
-      pickup = lookup_pickup_code(item['pickup'].split("|").first)
+      pickup = item['pickup_location_id'] ||= "299"
```

Comment:
@christinach why are you setting the item['pickup_location_id'] here?  I assume you really want a default not to modify the item `pickup = item['pickup_location_id'] || "299"` (notice no equal)

Reply:
my understanding is that the ||= will set the item['pickup_location_id'] to 299 if there is no value for the pickup_location_id.